Node operators: Using Visor (#457)
vegaprotocol/documentation: docs/node-operators/get-started/setup-validator.md

YOUR_VEGA_HOME_PATH/config/node/config.toml
```

> ## Initialise Visor for protocol upgrades

Reviewer: ```suggestion
## Initialise Visor (for smooth protocol upgrades)
```

maybe?

Reviewer: Maybe without smooth. But the protocol upgrade this way sounds a bit confusing. Or maybe it's just me.

Visor manages protocol upgrades, allowing the nodes running a network to automatically update to the latest version of the Vega protocol, without requiring manual intervention. Using Visor is optional, but recommended. 

> Read more about protocol upgrades (link)

Reviewer: Is the link missing?

Author: Yes it is - thank you :) Forgot to add it in after creating the page.

> Read more about protocol upgrades (link)

Author: ```suggestion
Read more about how to [propose a protocol upgrade](../how-to/propose-protocol-upgrades.md).
```

## Synchronise your node
You can either start the node from a snapshot (recommended, particularly for a long chain), or replay the full chain history to get up-to-date. If using Visor for protocol upgrades, follow the instructions below to start the node with Visor.

### Start node from a snapshot
[Snapshots](../how-to/use-snapshots.md): Use a recent network snapshot to start your node without having to replay the entire chain.

After you choose the snapshot you're starting from: 

**If you're using Visor**, start your node by running the service manager of your choice and use the following command:

```shell
visor run --home "VISOR_HOME_PATH"
```

**If you're not using Visor**, start your node by running the following command e.g. for sandbox:

```shell
vega start --home="YOUR_VEGA_HOME_PATH" --nodewallet-passphrase-file="YOUR_PASSPHRASE_FILE_PATH" --network=sandbox
```

### Replay from genesis
To replay all history from genesis: 

You can set a genesis file when starting the node with the following command, e.g. for sandbox:

```shell
vega start --home="YOUR_VEGA_HOME_PATH" --nodewallet-passphrase-file="YOUR_PASSPHRASE_FILE_PATH" --network=sandbox
```

> If using Visor, start your node by running the service manager of your choice, including the required args (flags) for network, etc, and use the following command:

Reviewer: "If using Visor, start your node by running the service manager of your choice" - > "If using Visor, start the Visor by running a service manager of your choice and do not forget to configure it."

Something like that makes more sense... Not the exact words.. :) Because the text above kind of give me feeling that I should start the Vega node with service manager and then run the Visor command.

> If using Visor, start your node by running the service manager of your choice, including the required args (flags) for network, etc, and use the following command:

Author: ```suggestion
If using Visor, start your node by running the service manager of your choice, configured with the required args (flags) for network, etc, and use the following command:
```

Author: @karlem How about this change?
